app.js

```diff
@@ -82,3 +82,41 @@ function hanldRemove(e) {
 }
 
 bookStore.addEventListener('click', hanldRemove);
+
+const headerDate = document.querySelector('.header-date');
+headerDate.innerHTML = Date();
+
+const bookSection = document.querySelector('.book-section');
+const addBookSection = document.querySelector('.add-section');
```

Comment:
- [ OPTIONAL ] To make the user experience better I would suggest you add an on click which sends you to the list of books page after the user has created a book.

Reply:
If we had to make an on-click event to send the user to the book list, what if the user has to add many books?  My partner and I see that it will be a little bit frustrating the fact that the user will be obliged to come back again to the add book section. That's why the user will have the possibility to type in as many books as he/she can and after finishing typing, can go check them in the book collection.
@GzimAsani

Reply:
@christianbiring1 you have made a great point :+1: